lib/views/backend_v1.2/spree/admin/payments/source_forms/_paypal_braintree.html.erb

```diff
@@ -0,0 +1,16 @@
+<% id = payment_method.id %>
+
+<fieldset class="no-border-bottom braintree-hosted-fields" data-braintree-hosted-fields data-id="braintree" data-payment-method-id="<%= id %>">
```

Comment:
Sure. It's a significant difference for Solidus version compatibility, but all the more reason to give it a named method.